MobileWallet/TariLib/Tor/OnionManager.swift

```diff
@@ -221,6 +233,7 @@ public class OnionManager: NSObject {
                         var completeObs: Any?
```

Comment:
Not clear if Obs is short for something...my first guess is Onion Bootstrap Status? If it's initials then `completeOBS` might be better but still unclear.

Reply:
Ah it observer.

Reply:
I don't think `Observer` is too long.

Reply:
and it looks like neither `completeObs` or `progressObs` are used, so just ditch them maybe (_does XCode warn that the return value is not used?_).

Reply:
This class was initially borrowed from the onion browser, so that stayed behind. Have renamed them. They are both used though, there's this method: `self.torController?.removeObserver(completeObserver)`

Reply:
Sorry didn't see the use first time. Cool!